lib/cli/utils.js

```diff
@@ -138,13 +138,22 @@ function runCommandChecks(
   requiredTemplateOptions,
   options,
   firmIdDefault,
-  partnerSupported = true
+  partnerSupported = true,
+  messageRequired = false
 ) {
-  if (!partnerSupported && options.partner) {
-    consola.error(
-      `The option "--partner" is not supported when using "--export-file"`
-    );
-    process.exit(1);
+  if (options.partner) {
+    if (!partnerSupported) {
+      consola.error(
+        `The option "--partner" is not supported when using "--export-file"`
+      );
+      process.exit(1);
+    }
```

Comment:
@Toby-Masters-SF maybe add a comment that we will need to remove this once Export Files are supported.